File: kattis/leftbeehind.py
a, b = list(map(int, input().split()))
while a != 0 or b != 0:
    if a + b == 13:
        print("Never speak again.")
    elif a == b:
        print("Undecided.")
    elif a > b:
        print("To the convention.")
    else:
        print("Left beehind.")
    a, b = list(map(int, input().split()))

File: kattis/leftbeehind.rb
a, b = gets.chomp.split(' ').map(&:to_i)
while a != 0 || b != 0 do
    if a + b == 13
        puts 'Never speak again.'
    elsif a == b
        puts 'Undecided.'
    elsif a > b
        puts 'To the convention.'
    else
        puts 'Left beehind.'
    end
    a, b = gets.chomp.split(' ').map(&:to_i)
end